src/iblt.h

```diff
@@ -69,12 +69,18 @@ class HashTableEntry
 class CIblt
 {
 public:
+    // Default constructor builds a 0 size IBLT, so is meant for two-phase construction.  Call resize() before use
     CIblt();
+    // Pass the expected number of entries in the IBLT table.  If the number of entries exceeds
+    // the expected, the false positive rate will rise
```

Comment:
Small thing; technically it's the decode failure rate that will increase. So I would rephrase as follows

`// Pass the expected number of entries in the IBLT table.  If the number of entries exceeds`
`// the expected, then the decode failure rate will increase dramatically.`